### DISCUSSION TITLE
Map generation problem

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#8071] [Q&A] [answered]
Hi ! When I try to regenerate maps (for #8502) I have this error ? Any idea why ?

![image](https://github.com/organicmaps/organicmaps/assets/116185937/2808fee4-db80-4020-8ea1-5bad5e3c6624)

**Comment by ghost** [comment]
Someone have an idea why I have this ? Maybe I forgot something.

**Comment by hb0nd** [comment, marked as answer]
make sure `OMIM_PATH` in the ini file is set correctly, e.g., my value is `/home/harry/Documents/Git/organicmaps`

**Reply by ghost** [reply]
Thanks ^^.
